rails/getting-started/existing.html.md

```diff
@@ -85,6 +85,15 @@ This shows the past few log file entries and tails your production log files.
 
 It can be helpful to open a Rails console to run commands and diagnose production issues.
 
+To do this, we will [login with SSH](/docs/flyctl/ssh/) to our
+application VM. There is a one-time setup tasks for using SSH. Follow the instructions.
+
+```
+fly ssh issue --agent
+```
+
+With SSH configured, let's open a console.
+
```

Comment:
@bradgessler I added a section to the rails guides, to skip a foot gun people might hit on their first try.